stdlib/array.gr

```diff
@@ -769,6 +852,9 @@ provide let unique = array => {
  * @returns The new array containing indexed pairs of `(a, b)`
  *
  * @throws Failure(String): When the arrays have different sizes
+ * 
+ * @example
+ * Array.zip([> 1, 2, 3], [> 11, 12, 13]) // [> (1, 11), (2, 12), (3, 13)]
```

Comment:
Same As Above:
```suggestion
 * Array.zip([> 1, 2, 3], [> 11, 12, 13]) == [> (1, 11), (2, 12), (3, 13)]
```

Reply:
Same comment as before. This is not returning void so simply in line the result.....